.travis.yml

```diff
@@ -60,23 +47,13 @@ matrix:
       rust: stable
       script:
         - cargo run --manifest-path ci/remove-dev-dependencies/Cargo.toml */Cargo.toml
-        # Check default-features
         - cargo build --all
-        # Check compat & thread-pool & async-await features
```

Comment:
It should be sufficient to check these features on `1.36.0 build` and `features check` (also async-await feature is activated by default).